### ISSUE TITLE
New Implementations 

### ISSUE THREAD

**Issue by kotik1998** [ThunbergOlle/SteamTradeBot#9]
Hi there again!
Can you implement, multi user support ( acc1, acc2, acc3 ....), and having a online/offline button for each bot on the main page where you can be online or be offline. like this https://www.w3schools.com/howto/tryit.asp?filename=tryhow_css_switch

**Comment by ThunbergOlle** [comment]
Yes, I will add that on the list! I think I will have to rewrite some code but hopefully, I can publish a somewhat stable result by the end of October. 👍 

I created a testing branch so you can see the progress without it affecting the main branch: https://github.com/ThunbergOlle/SteamTradeBot/tree/multiple-accounts-support

Thanks for the idea! ❤️
